README.md

```diff
@@ -124,3 +124,39 @@ operator-sdk test local ./test/e2e --up-local --kubeconfig=$KUBECONFIG --namespa
 ```
 
 You can also run the tests with `make e2e-test`.
+
+## Uninstalling an Operator from the CLI
+
+After an operator has been installed from marketplace, to uninstall the operator from a particular OperatorGroup you need to delete the following resources. Below we are using the descheduler operator as an example:
+
+1. Delete the installation `CatalogSourceConfig` in the `openshift-marketplace` that contains the installed operator package. If created via the OperatorHub UI, it will be named `installed-<OPERATORSOURCE>-<TARGETNAMESPACE>` based on the `OperatorSource` that the operator comes from and the `targetNamespace` that the operator was installed to.
+```bash
+    $ oc delete catalogsourceconfig installed-community-openshift-operators -n openshift-marketplace
+```
+If other operators were installed with the same `CatalogSourceConfig`, to uninstall only one of the operators use the following command to modify the `spec.packages` field and remove the operator's packageName from the comma-separated string:
+```bash
+    $ oc edit catalogsourceconfig installed-community-openshift-operators -n openshift-marketplace
+```
+This will open up your terminal's default editor. Look at the `spec` section of the `CatalogSourceConfig`:
+```bash
+    spec:
+      csDisplayName: Community Operators
+      csPublisher: Community
+      packages: descheduler,jaeger
+```
+Remove `descheduler` and the following comma:
+```bash
+    spec:
+      csDisplayName: Community Operators
+      csPublisher: Community
+      packages: jaeger
+```
+Save the change and the marketplace-operator will reconcile the `CatalogSourceConfig`.
+2. Delete the `Subscription` in the `targetNamespace` that the operator was installed into. If created via the OperatorHub UI, it will be named after the operator's packageName.
```

Comment:
Please account for how the subscription was created in `Installing an operator using Marketplace`